src/providers/ArticleProvider.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import Articles from "../pages/Articles";
+import SingleArticle from "../pages/SingleArticle";
+import axios from "axios";
+
+function ArticleProvider() {
+const [articles, setArticles] = useState([]);
+useEffect(() => {
+    axios.get(`https://nc-news-project-cf.herokuapp.com/api/articles`)
+    .then((response) => {
+        setArticles(response.data.articles)
+    })
+}, [])
+
+return (
+    <div>
+        <Articles articles={articles}/>
+        <SingleArticle />
```

Comment:
This wont work for the next ticket, as you'll have the fill list, as well as a singleArticle (when it has some content). 
i'd remove this single article.